dev/bin/manager-run.sh

```diff
@@ -48,7 +48,7 @@ mvn \
   -Dmanaged-connectors.kafka.client.id=${mc_client_id} \
   -Dmanaged-connectors.kafka.client.secret=${mc_client_secret} \
   -Dmanaged-connectors.kafka.security.protocol=SASL_SSL \
-  -Dmanaged-connectors.services.url=https://cos-fleet-manager-cos.rh-fuse-153f1de160110098c1928a6c05e19444-0000.eu-de.containers.appdomain.cloud \
```

Comment:
If this is a new parameter required by the script could you update the header of the file 
```
########
# Run Fleet Manager locally in dev mode
#
# Env vars:
```

and the dev/README.md?